docs/robots/warthog/user_manual_warthog.mdx

```diff
@@ -4,7 +4,671 @@ sidebar_position: 2
 ---
 
 import ComponentIntroductionWarthog from "/components/introduction_warthog.mdx";
+import Support from "/components/support.mdx";
+
+## Introduction
 
 <ComponentIntroductionWarthog />
 
-TODO
+### What's Included
+
+Each Warthog includes:
+
+- 1X Warthog UGV
+  - Onboard computer
+  - User Panel with power, Ethernet, Serial(RS232) and USB connectivity
+  - 48V Lead Acid Battery Pack
+  - Battery Charger
+- 1X Futaba Remote Control (R/C)
+- 1X Warthog User Manual
```

Comment:
When we go live with these docs, we won't be including the User Manual any more right? More likely just a 1-page quick start guide.

Reply:
Agreed, this should get updated.